packages/material-ui-naked/src/Dialog/Dialog.js

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils/helpers';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Fade from '@material-ui/core/Fade';
+import Paper from '@material-ui/core/Paper';
```

Comment:
If an unstyled component imports other (unstyled?) components, how does the style for that imported component get applied by the wrapper component.

Reply:
Correct, we would need to progressively change the imports as we migrate the components.

Reply:
That wasn't the question. If an unstyled component imports another unstlyed component, how does that unstyled subcomponent get styled by the user?

Reply:
He would have to use the global classes or recreate the whole dependency chain with the factory method. Something we would have to do in core.